be/src/service/service_cn/starrocks_cn.cpp

```diff
@@ -73,7 +73,7 @@ void start_cn() {
 
     // 3. Start http service.
     std::unique_ptr<starrocks::HttpServiceCN> http_service = std::make_unique<starrocks::HttpServiceCN>(
-            exec_env, starrocks::config::webserver_port, starrocks::config::webserver_num_workers);
+            exec_env, starrocks::config::be_http_port, starrocks::config::be_http_num_workers);
```

Comment:
You can have a look of `common/configbase.cpp`
```
        // compatible with doris_config
        key = std::regex_replace(key, doris_start, "");
```
Like this place, you need to replace the webserver_port with be_http_port .So they it can be compatible with the old cluster .